.gitignore

```diff
@@ -9,6 +9,7 @@
 /spec/reports/
 /stdout.txt
 /tmp/
+/unwrappr-*.gem
```

Comment:
I don’t see a reason to store any gems in the repo. Perhaps a simple `*.gem` would suffice.